src/main/java/org/example/src/game/Game.java

```diff
@@ -70,12 +58,31 @@ protected static Unit whoWin(){
         else return enemy;
     }
 
+    protected static void gameLoop(){
+        while(hero.isAlive()){
+            setEnemy(OpponentSpawner.spawn());
+            fight();
+            if(whoWin()==hero){
+                hero.levelUp();
+                System.out.println(
+                        "\n\t|Excelent +1 Level|"+
+                        "\n  ==============================="+
+                        "\n===== !!! Start next fight !!! ===="+
+                        "\n  ===============================\n"
+                );
+            }
+            else{
+                System.out.println("\n\tGAME OVER");
+                return;
+            }
+        }
+    }
+
     public static void start(){
         System.out.println("\nNazwij bohatera:");
```

Comment:
Cosmetic change, decide to use one language in this implementation - either Polish or English.